tools/autograd/gen_python_functions.py

```diff
@@ -57,11 +79,11 @@
 PY_VARIABLE_METHOD_DEF = CodeTemplate("""\
 {"${name}", (PyCFunction)${pycname}, ${flags}, NULL},""")
 
-UNPACK_ARG = CodeTemplate("""\
-${formal_arg} = ${actual};""")
-
 UNPACK_SELF = "auto& self_ = reinterpret_cast<THPVariable*>(self)->cdata;"
 
+FUNCTION_PROTOTYPE = CodeTemplate("""\
```

Comment:
I think in C++ this would be considered closer to a function signature since it doesn't have a return type.

Reply:
This isn't quite a C++ function prototype or a C++ type signature. The names and default values are important here, which isn't true of C++ signatures.

Mostly I've been calling these things prototypes since we're using "signature" to refer to the string we compute to find matching declarations in Declarations.yaml.